src/components/ListingContactUs/ListingContactUs.test.js

```diff
@@ -22,12 +22,12 @@ describe('ListingContactUs component', () => {
   it('should renter tabs', () => {
     expect(component.querySelector('.listing-contact-us__tabs')).toBeInTheDocument();
   });
-  it('should change tabs content', () => {
-    fireEvent.click(getByTextFunc('Schedule a Visit'));
-    // Change it when the form component will be done
-    expect(getByTextFunc('Schedule a Visit Form Component')).toBeInTheDocument();
-    fireEvent.click(getByTextFunc('Contact Us'));
-    // Change it when the form component will be done
-    expect(getByTextFunc('Contact Us Form Component')).toBeInTheDocument();
-  });
```

Comment:
Do we need this test?

Reply:
I will need it. But instead of checking a text here will be a form existing checking. That is because I commented it